vendor/sources/main.tf

```diff
@@ -18,11 +18,11 @@ locals {
       vm_size                       = "Standard_E8s_v3"
       enable_accelerated_networking = false
       data_disks_configuration = {
-        disks_type       = "Premium_LRS,Premium_LRS,Premium_LRS,Premium_LRS,Premium_LRS,Premium_LRS,Premium_LRS"
-        disks_size       = "128,128,128,128,128,128,128"
-        caching          = "ReadOnly,ReadOnly,None,None,ReadOnly,ReadOnly,ReadOnly"
-        writeaccelerator = "false,false,false,false,false,false,false"
-        luns             = "0,1#2,3#4#5#6#7"
+        disks_type       = "Premium_LRS,Premium_LRS,Premium_LRS,Premium_LRS,Premium_LRS,Premium_LRS,Premium_LRS,Premium_LRS"
```

Comment:
Whats the last disk added for?

Reply:
@peteratsuse confirmed me that the luns configuration is `"0,1#2,3#4#5#6#7"`, so this asks for 8 disks